lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -45,23 +46,36 @@ class EstimatedInputLatency extends Audit {
       throw new LHError(LHError.errors.NO_FMP);
     }
 
-    const latencyPercentiles = TracingProcessor.getRiskToResponsiveness(tabTrace, startTime);
-    const ninetieth = latencyPercentiles.find(result => result.percentile === 0.9);
-    const rawValue = parseFloat(ninetieth.time.toFixed(1));
+    const events = TracingProcessor.getMainThreadTopLevelEvents(tabTrace, startTime)
+      .filter(evt => evt.duration >= 1);
```

Comment:
since a blank trace gets a value of 16, can't we safely remove any events with a duration < 16?

Reply:
well a trace with constant 16ms events would technically get a EQT of 24ms since we arbitrarily add 16ms to whatever value we compute, I'm all for doing more filtering here though, who cares about your small events